package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mwp-cli",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
The versioning doesn't happen here, it happens in the `Makefile` at the root of the repo. 

This line:
```VERSION ?= 8.3.$(CI_BUILD_NUMBER)```

Just increment the `3` -> `4`